### PR TITLE
Css final fix

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by LauraCourquin — DISMISSED** [review]
It's good for me !

File: frontend/src/components/CardEmployerInfo/style.jsx [hunk 1]
```diff
@@ -137,56 +159,55 @@ export default styled.div`
...
-    background-image: url(${editLogo});
-    background-size: cover;
-    background-color: rgb(255, 255, 255, 0.8);
-    border: none;
-    position: absolute;
-    right: 0.5rem;
-    top: 0.5rem;
-    width: 2rem;
-    height: 2rem;
+  .form {
```
Comment (LuigiMenez) [inline frontend/src/components/CardEmployerInfo/style.jsx]: Why double display flex?

File: frontend/src/components/CardProfileInfo/index.jsx [hunk 2]
```diff
@@ -129,9 +99,39 @@ function CardProfileInfo() {
...
+        navigate("/");
       });
   };
+
+  const [isFollow, setIsFollow] = useState(false);
+  const handleIsFollow = () => {
+    setIsFollow(!isFollow);
+  };
+
+  const customStyles = {
```
Comment (LuigiMenez) [inline frontend/src/components/CardProfileInfo/index.jsx:113]: Why isn't the style in the style file?

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Fix minor

File: frontend/src/components/CardEmployerInfo/style.jsx (hunk 1 above)
Comment (ghost) [reply frontend/src/components/CardEmployerInfo/style.jsx]: error

File: frontend/src/components/CardProfileInfo/index.jsx (hunk 2 above)
Comment (ghost) [reply frontend/src/components/CardProfileInfo/index.jsx:113]: Because it's a package and we cannot put this in the style file, i tried

**Review by LauraCourquin — DISMISSED** [review]
It's good for me !

**Review by LuigiMenez — DISMISSED** [review]
Ok for me.

**Review by LoicBrassart — DISMISSED** [review]
SnE, except for the Modal's styled

**Comment by ghost** [comment]
Fix conflict with dev

**Review by LauraCourquin — DISMISSED** [review]
It's good for me !

**Review by LauraCourquin — APPROVED** [review]
OK

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**